core/src/shred_fetch_stage.rs

```diff
@@ -24,7 +23,9 @@ use {
     },
 };
 
-const DEFAULT_LRU_SIZE: usize = 10_000;
+const DEDUPER_FALSE_POSITIVE_RATE: f64 = 0.001;
```

Comment:
what is K if thats the false positive rate?  I think you want to constrain K instead of the rate when creating the bloom filter, otherwise it will pick a pretty high K.